server/api/service_middleware_test.go

```diff
@@ -51,23 +51,27 @@ func (s *testServiceMiddlewareSuite) TearDownSuite(c *C) {
 func (s *testServiceMiddlewareSuite) TestConfigAudit(c *C) {
 	addr := fmt.Sprintf("%s/service-middleware/config", s.urlPrefix)
 	ms := map[string]interface{}{
-		"enable-audit": "true",
+		"enable-audit":      "true",
+		"enable-rate-limit": "true",
 	}
 	postData, err := json.Marshal(ms)
 	c.Assert(err, IsNil)
 	c.Assert(tu.CheckPostJSON(testDialClient, addr, postData, tu.StatusOK(c)), IsNil)
 	sc := &config.ServiceMiddlewareConfig{}
 	c.Assert(tu.ReadGetJSON(c, testDialClient, addr, sc), IsNil)
 	c.Assert(sc.EnableAudit, Equals, true)
+	c.Assert(sc.EnableRateLimit, Equals, true)
 	ms = map[string]interface{}{
 		"audit.enable-audit": "false",
+		"enable-rate-limit":  "false",
 	}
 	postData, err = json.Marshal(ms)
 	c.Assert(err, IsNil)
 	c.Assert(tu.CheckPostJSON(testDialClient, addr, postData, tu.StatusOK(c)), IsNil)
 	sc = &config.ServiceMiddlewareConfig{}
 	c.Assert(tu.ReadGetJSON(c, testDialClient, addr, sc), IsNil)
 	c.Assert(sc.EnableAudit, Equals, false)
+	c.Assert(sc.EnableRateLimit, Equals, false)
```

Comment:
Could you please separate the test for rate-limit from `audit`?

Reply:
> Meanwhile, could you please add some tests of updating the item of `limiter-config`?

Some tests will be added in #4843 after providering update API

Reply:
> Meanwhile, could you please add some tests of updating the item of `limiter-config`?

I add a simple test, more test will in #4843 by specific API